### PR TITLE
Bump version to 0.2.19

### PR DESCRIPTION
**Pull request by zanieb**

### CODE REVIEW

**Review by charliermarsh — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+
+- Add `--disable-pip-version-check` to compatibility arguments ([#4672](https://github.com/astral-sh/uv/pull/4672))
+- Allow `uv pip sync` to clear an environment with opt-in ([#4517](https://github.com/astral-sh/uv/pull/4517))
+- Add `--invert` to `uv pip tree` ([#4621](https://github.com/astral-sh/uv/pull/4621))
+- Omit `(*)` in `uv pip tree` for empty packages ([#4673](https://github.com/astral-sh/uv/pull/4673))
+- Add `--package` to `uv pip tree` ([#4655](https://github.com/astral-sh/uv/pull/4655))
+
+### Bug fixes
+
+- Fix implementation of `GitDatabase::contains` ([#4698](https://github.com/astral-sh/uv/pull/4698))
```
Comment (charliermarsh) [inline CHANGELOG.md:19]: Maybe a description of the bug itself rather than the implementation?

Comment (zanieb) [reply CHANGELOG.md:19]: Yeah want to change that when you add https://github.com/astral-sh/uv/pull/4740?

Comment (zanieb) [reply CHANGELOG.md:19]: (I presume you're going to sneak that in haha)

**Comment by zanieb** [comment]
I tested a macOS wheel and #4632 worked nicely.

**Pull request merged**